src/libraries/System.Linq/tests/MaxTests.cs

```diff
@@ -11,13 +11,16 @@ public class MaxTests : EnumerableTests
     {
         public static IEnumerable<object[]> Max_AllTypes_TestData()
         {
-            for (int length = 2; length < 33; length++)
+            for (int length = 2; length < 65; length++)
             {
                 yield return new object[] { Shuffler.Shuffle(Enumerable.Range(length, length).Select(i => (byte)i)), (byte)(length + length - 1) };
                 yield return new object[] { Shuffler.Shuffle(Enumerable.Range(length, length).Select(i => (byte)i).ToArray()), (byte)(length + length - 1) };
 
-                yield return new object[] { Shuffler.Shuffle(Enumerable.Range(length, length).Select(i => (sbyte)i)), (sbyte)(length + length - 1) };
-                yield return new object[] { Shuffler.Shuffle(Enumerable.Range(length, length).Select(i => (sbyte)i).ToArray()), (sbyte)(length + length - 1) };
+                // Unit Tests does +T.One so we should generate data up to one value below sbyte.MaxValue
+                if ((length + length) < sbyte.MaxValue) {
```

Comment:
This predates your own changes, but curious why this uses `length + length` instead of `2 * length`.

Reply:
I used (length + length) as this is how it´s done during test data generation, so it uses the same style to check against this. It´s easier to spot like that / mentally link it to the generated test data i figured.